src/sedcli_main.c

```diff
@@ -739,6 +766,53 @@ static int handle_setpw(void)
 	return ret;
 }
 
+static int handle_listlr(void)
+{
+	struct sed_device *dev = NULL;
+	int ret;
+	struct sed_opal_lockingrange_arr lrs;
+
+	memset(&lrs, 0, sizeof(struct sed_opal_lockingrange_arr));
+
+	sedcli_printf(LOG_INFO, "Enter Admin1 password: ");
+
+	ret = get_password(&opts->pwd);
+
+	if (ret != 0) {
```

Comment:
if (ret) should be fine.
I know we have this if(ret != 0) condition through out the code, which can be changed in the future.